deploy/charts/cert-manager/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,28 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.prometheusrule.enabled }}
+---
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "cert-manager.fullname" . }}
+{{- if .Values.prometheus.prometheusrule.namespace }}
+  namespace: {{ .Values.prometheus.prometheusrule.namespace }}
+{{- else }}
+  namespace: {{ .Release.Namespace | quote }}
+{{- end }}
+  labels:
+    app: {{ include "cert-manager.name" . }}
+    app.kubernetes.io/name: {{ include "cert-manager.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "controller"
```

Comment:
This label can be removed as it applies to the deployment

Reply:
Only the `app.kubernetes.io/component` label or should all labels be removed?

Reply:
I think only the `app.kubernetes.io/component` label should be removed.